src/blackdoor/cqbe/node/Node.java

```diff
@@ -186,18 +188,20 @@ public void setPort(int port) {
 		 */
 		public void setStorageDir(String storageDir) {
 			this.storageDir = storageDir;
-			config.put("storageDir", storageDir);
+			config.put("storage_directory", storageDir);
 		}
 
 		/**
-		 * Sets the settings file directory Passed from DART.Join
+		 * Sets the settings file Passed from DART.Join
 		 * 
 		 * @param directory
 		 *            - Directory to be used by created node
 		 */
-		public void setSettings(String settingsDir) {
-			loadSettings("settingsDir");
+		public void setSettings(String filename) {
+			config.loadSettings(new File(filename));
```

Comment:
thats what this does. CLI calls the the node builder constructor, which loads the default config. If the setting flag is set, it will call this function and load the user's settings from file.